applications/nrf_desktop/configuration/nrf52840_pca10059/app_ZDebug.conf

```diff
@@ -140,6 +140,7 @@ CONFIG_BT_GATT_HIDS_C=y
 CONFIG_BT_GATT_HIDS_C_REPORTS_MAX=9
 
 CONFIG_BT_LL_NRFXLIB=y
+CONFIG_BLECTRL_MAX_CONN_EVENT_LEN=2500
```

Comment:
Since you have 2 connections you might want to consider 3750 instead of 2500

Reply:
We also use peripherals with 1 ms connection interval and we want to achieve high report rate for them. Increasing this parameter would reduce report rate for this sort of devices when connected together with BLE devices.

Reply:
Maybe we could set this value to 3000 (to make it multiple of the shorter connection interval). This should not have negative impact on "quicker" device (with current values connection events overlap).

Reply:
AFAIK this parameter would not affect the LLPM connection. @rugeGerritsen Any comment on this?

Reply:
The BLE event length will affect the LLPM connection for the case where there is 1 LLPM connection and 1 BLE connection. The longer the BLE event length, the more consequent LLPM connection events that will be kicked out by the scheduler.

Reply:
It does not directly affect LLPM, but the longer BLE connection event is, the more LLPM packet rate is lowered. BLE occupies the link for longer time, so LLPM is unable to send data for longer time.

Reply:
Ok. I didn't know they were used together. Then 2500 makes most sense. Yes.